custom_components/tuya_local/devices/klarstein_wonderwall_air.yaml

```diff
@@ -0,0 +1,118 @@
+name: Klarstein Wonderwall Air
+products:
+  - id: cl2wgyhohpjjjuzu
+    name: Wonderwall Art Smart
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 10
+      range:
+        min: 0
+        max: 370
+      unit: "\u00b0C"
+    - id: 3
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 10
+          step: 10
+      range:
+        min: 0
+        max: 500
+      unit: "\u00b0C"
+    - id: 4
+      type: string
+      mapping:
+        - dps_val: manual
+          value: Manual
+        - dps_val: holiday
+          value: Holiday
+        - dps_val: auto
+          value: Auto
+      name: preset_mode
+secondary_entities:
+  - entity: number
+    name: Timer
+    icon: "mdi:timer-sand"
+    category: config
+    dps:
+      - id: 10
+        type: integer
```

Comment:
Is it really an integer?  Most devices that split timer set and countdown reporting use a string for setting (reported in the iot.tuya.com docs as enum type), even if the contents of the string are always numbers (though often 0 is replaced by "cancel").

Reply:
Modified to match klarstein_wonderwall_irheater.yaml